R/data.R

```diff
@@ -83,4 +83,39 @@ NULL
 #'
 NULL
 
-
+#' @title SilvermanAGutData
```

Comment:
We can omit "@title" - the first row is interpreted as title. This is how we do it most of the time.

Reply:
I will change those, but I just followed other formatting of other datsets in miaTime (tehy ahve title and description